config.md

```diff
@@ -325,6 +326,11 @@ calls.
 
 A boolean that enables/disables inlay-hints for deduced types.
 
+### Designators
+{:.v14}
+
+A boolean that enables/disables inlay-hints for array designators. eg: `std::vector<int> arr = {[0]= 1, [1]= 2}` (`Designators: true`). `std::vector<int> arr = {1, 2}` (`Designators: false`)
```

Comment:
this also disables aggregate initializers (e.g. `Foo f{/*a*/1, /*b*/2};`) could you update the documentation to cover both?

Reply:
All right. I'm working on it.

Reply:
> this also disables aggregate initializers (e.g. `Foo f{/*a*/1, /*b*/2};`) could you update the documentation to cover both?

Does it? it seemed doesn't have effect from my screenshot.
![image](https://user-images.githubusercontent.com/52443370/215271450-3971aa4f-229b-4ef9-97c2-58f7bef1807e.png)
![image](https://user-images.githubusercontent.com/52443370/215271454-b0412bf8-655a-4fde-bd98-c27a874d365e.png)

Reply:
Since you have a constructor, you're actually calling it rather than having a designated initializer. If you drop the constructor `Foo(int, bool);` and construct it using aggregate initialization, you should see inlay hints disappearing.